app/models/vehicle.rb

```diff
@@ -1,3 +1,8 @@
 class Vehicle < ApplicationRecord
   belongs_to :user
+  validates :category, inclusion: { in:
+    ["trottinette", "trottinette électrique", "vélo", "vélo électrique",
```

Comment:
ok nice categories, in english you can write : motorbike, electric motorbike, bicycle, electric bicycle, scooter, electric scooter, kick scooter, electric kick scooter

Reply:
thanks perfect i will change it in english